### PR TITLE
[bitnami/contour] Allow extra arguments to be specified, and passed to Contour container

### PR DESCRIPTION
**Pull request by MrSaints**
**Description of the change**

Allow extra arguments to be specified, and passed to Contour container.

It can be specified in the values using the `contour.extraArgs` key. An array must be provided.

**Benefits**

This is useful for enabling the PROXY protocol by specifying
`--use-proxy-protocol` as an extra arg.

See https://projectcontour.io/guides/proxy-proto/ for more information.

It can potentially be used for other options.

**Possible drawbacks**

- The implementation requires hard coding / specifying the YAML indentation
- It does not account for removal of existing arguments in the template (a full arg override option is probably needed for this, but it is out-of-scope for this change)

**Applicable issues**

N/A

**Additional information**

Other options considered:

1. Full arg override: goes against sane defaults, and may be too fiddly
2. Boolean option / toggle: not extensible if we want to cover other non-default options

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/contour/templates/contour/deployment.yaml [hunk 1]
```diff
@@ -67,6 +67,9 @@ spec:
             {{- if .Values.ingressClass }}
             - --ingress-class-name={{ .Values.ingressClass }}
             {{- end }}
+            {{- if .Values.contour.extraArgs }}
+            {{ toYaml .Values.contour.extraArgs | indent 12 | trim }}
```
Comment (dani8art) [inline bitnami/contour/templates/contour/deployment.yaml:71]: Maybe it could be even better using `common.tplvalues.render` helper, don't you think?

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi @MrSaints thank you so much for this PR

Please, take a look at my comment

File: bitnami/contour/templates/contour/deployment.yaml (hunk 1 above)
Comment (MrSaints) [reply bitnami/contour/templates/contour/deployment.yaml:71]: Sounds good! Thanks for the feedback.

**Comment by MrSaints** [comment]
Rebased due to upstream changes.

**Review by dani8art — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**